maboss/ensemble/ensemble.py

```diff
@@ -131,7 +132,16 @@ def run(self, workdir=None, overwrite=False, prefix="res"):
         # return EnsembleResult(self.models_files, self._cfg, "res", self.individual_results, self.random_sampling)
 
     def set_models_path(self, path):
```

Comment:
Could this method be called from the constructor, for avoiding code duplication?

Reply:
Indeed